### DISCUSSION TITLE
Blacklist some themes from the random ones choosen

### DISCUSSION THREAD

**Discussion by molitona** [ohmyzsh/ohmyzsh#10664] [Help] [answered]
Hi, I've my zsh theme var set to RANDOM, some themes are very slow to show prompt. How can I blacklist them ??

**Comment by carlosala** [comment, marked as answer]
You have the answer in the README.md file!
For example with two themes that you don't want to appear:
`ZSH_THEME_RANDOM_IGNORED=(pygmalion tjkirch_mod)`

**Reply by mcornella** [reply]
Also documented in the [Settings wiki section](https://github.com/ohmyzsh/ohmyzsh/wiki/Settings#random-theme).
